splash/browser_tab.py

```diff
@@ -1116,6 +1185,140 @@ def log(self, message, min_level=None):
         log.msg(message, system='render')
 
 
+class ElementsStorage(QObject):
```

Comment:
Could you please add docstrings for these classes (ElementsStorage, Event, EventHandlersStorage, EventsStorage)?